test/local/metrics/context.js

```diff
@@ -905,7 +897,7 @@ describe('metricsContext', () => {
     () => {
       var expectedTime = 1451566800000
       var expectedSalt = '4d6f7a696c6c6146697265666f782121'
-      var expectedHmac = 'c89d56556d22039fbbf54d34e0baf206'
+      var expectedHmac = '2a204a6d26b009b26b3116f643d84c6f'
```

Comment:
Even though this and the preceding test are failing cases, I updated the expected signature to make sure that they're failing for the right reason.